aws-lc-sys/builder/cmake_builder.rs

```diff
@@ -176,7 +175,21 @@ impl CmakeBuilder {
             emit_warning(&format!(
                 "CMAKE_TOOLCHAIN_FILE environment variable set: {toolchain}"
             ));
+            if target_env() == "ohos" {
+            // The Open Harmony toolchain requires the following tweaks
+                cmake_cfg.define("OHOS_ARCH", target_arch());
```

Comment:
The expected values from the toolchain file for the cache variable seem to be one of
`arm64-v8a`, `armeabi-v7a` and `x86_64`, which doesn't directly match the rust target triples except for x86_64.

Reply:
Thanks for the info! I'll update this PR to have a better mapping of Rust target arch's to `OHOS_ARCH` values.